packages/date-picker/src/basic/time-spinner.vue

```diff
@@ -273,6 +274,14 @@
 
         this.modifyDateField(label, now);
         this.adjustSpinner(label, now);
+      },
+      amPm(hour) {
+        let shouldShowAmPm = this.amPmMode.toLowerCase() === 'a';
+        if (!shouldShowAmPm) return '';
+        let isCapital = this.amPmMode === 'A';
+        let content = (hour < 12) ? ' am' : ' pm';
+        if (isCapital) content = content.toUpperCase();
```

Comment:
Is it a good idea to change `isCapital` to `isUpperCase` to match the JavaScript function called `toUpperCase()`?

Reply:
I think it's fine to use === 'A', because amPmMode only have 2 options: ['a', 'A'].